### DISCUSSION TITLE
Bottleneck Finder - Need help 5800X 32GB 3600MHZ RAM 2x2TB Firecuda 520

### DISCUSSION THREAD

**Discussion by vvavepacket** [madMAx43v3r/chia-plotter#302] [Q&A]
I need help finding my bottleneck.. My friend who has the same setup of mine gets around 30 minutes for every plot...

Here is my spec:
5800X 8 Cores 16 Threads
Patriot Viper Steel DDR4 32GB (2 x 16GB) 3600MHz Kit - PVS432G360C8K
Seagate Firecuda 520 2TB Performance Internal Solid State Drive SSD PCIe Gen4 X4

Here are my tests across different threads
16 threads - 2834 seconds | 47.23333 min
8 threads - 3127.17 seconds | 52.1195 min
4 threads - 3244.51 seconds | 54 min 

You can see my fastest for 16 threads is 47 mins.. but my friend gets 30 mins..

Is there anything wrong in my system?

**Comment by Kiggles** [comment]
You say your friend has the same setup, but does your friend literally have the same components, model, capacities, etc?

**Reply by vvavepacket** [reply]
He has a different board brand but it shouldnt matter really as we have the same SSD and mobo support for both is Gen 4 NVME. He has 1TB though,,, which should make mine faster since I have a 2TB

**Reply by therealflinchy** [reply]
> He has a different board brand but it shouldnt matter really as we have the same SSD and mobo support for both is Gen 4 NVME. He has 1TB though,,, which should make mine faster since I have a 2TB

1tb usually faster than 2tb, but this drive does have more cache in 2tb, 1/3 of drive capacity.

that ssd also goes VERY slow once you fill the cache, like 500-600mbps or something.

does your friend have a model of SSD that maintains it's speed for longer? or doesn't drop as far?
